docs/modules/publisher-dmmigration.md

```diff
@@ -12,49 +22,43 @@ For this set of methods, the encoding is JSON and no compression or payload chun
+| **UnpublishAllNodes**            | EndpointUrl                                                     | Status, List\<StatusResponse\>                               | Yes          |
+| **GetConfiguredEndpoints**       | -                                                               | List\<EndpointUrl\>                                          | Yes          |
+| **GetConfiguredNodesOnEndpoint** | EndpointUrl                                                     | EndpointUrl, List< OpcNodeOnEndpointModel >    where OpcNodeOnEndpointModel contains:    Id ExpandedNodeId OpcSamplingInterval OpcPublishingInterval DisplayName HeartbeatInterval SkipFirst | Yes          |
+| **GetDiagnosticInfo**            | -                                                               | DiagnosticInfoMethodResponseModel                            | Yes          |
+| **GetDiagnosticLog**             | -                                                               | MissedMessageCount, LogMessageCount, List\<Log\>             | No*          |
+| **GetDiagnosticStartupLog**      | -                                                               | MissedMessageCount, LogMessageCount, List\<Log\>             | No*          |
+| **ExitApplication**              | SecondsTillExit (optional)                                      | StatusCode, List\<StatusResponse\>                           | No*          |
+| **GetInfo**                      | -                                                               | GetInfoMethodResponseModel >>    VersionMajor  VersionMinor  VersionPatch  SemanticVersion  InformationalVersion OS  OSArchitecture  FrameworkDescription | No*          |
+
+*This functionality is provided by the IoT Edge `edgeAgent` module via its own direct methods, see [this page](https://docs.microsoft.com/azure/iot-edge/how-to-edgeagent-direct-method) for more information.
 
 A [sample configuration application](https://github.com/Azure-Samples/iot-edge-opc-publisher-nodeconfiguration) as well as an [sample application for reading diagnostic information](https://github.com/Azure-Samples/iot-edge-opc-publisher-diagnostics) are provided from OPC Publisher leveraging this interface.
 
 _Please note that the samples applications are not actively maintained and may be outdated._
 
 ## Direct Methods of version 2.8.2
 
-Inherited direct methods from 2.5.x:
-
+The following set of direct method api is inherited from 2.5.x API set.
 - PublishNodes_V1
 - UnpublishNodes_V1
 - UnpublishAllNodes_V1
 - GetConfiguredEndpoints_V1
 - GetConfiguredNodesOnEndpoint_V1
 - GetDiagnosticInfo_V1
 
-New direct methods:
-
-- AddOrUpdateEndpoints_V1
-
-Please note that the recommended way of using the direct methods is with `_V1` suffix however, direct method without `_V1` suffix is also supported.
+Please note that in 2.8.2 the recommended way of using the direct methods is with `_V1` suffix however.
```

Comment:
I feel like `however` here belongs to the beginning of next sentence instead of being in the end of this one.

Reply:
indeed. fixed!